test/unit/host_test.rb

```diff
@@ -1367,6 +1367,10 @@ def teardown
       assert_equal hosts.count, 1
       assert_equal ["num001.example.com"], hosts.map { |h| h.name }.sort
 
+      hosts = Host::Managed.search_for("facts.memory_mb ~ 64498")
```

Comment:
You tried with postgres? It failed on my side before the fix
On Aug 24, 2015 2:27 PM, "Daniel Lobato García" <notifications@github.com>
wrote:

> In test/unit/host_test.rb
> <https://github.com/theforeman/foreman/pull/2626#discussion_r37741583>:
>
> > @@ -1367,6 +1367,10 @@ def teardown
> >        assert_equal hosts.count, 1
> >        assert_equal ["num001.example.com"], hosts.map { |h| h.name }.sort
> >
> > +      hosts = Host::Managed.search_for("facts.memory_mb ~ 64498")
>
> This test is passing on the current develop branch, without the change
> above in scoped_search_extensions. Could you add some test that tests the
> functionality added here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/theforeman/foreman/pull/2626/files#r37741583>.
>